pkg/loop/plugin.go

```diff
@@ -0,0 +1,27 @@
+package loop
+
+import (
+	"github.com/hashicorp/go-plugin"
+	"google.golang.org/grpc"
+
+	"github.com/smartcontractkit/chainlink-relay/pkg/logger"
+)
+
+type pluginClient struct {
+	atomicBroker
+	atomicClient
+	*lggrBroker
+}
+
+func newPluginClient(stopCh <-chan struct{}, lggr logger.Logger, broker *plugin.GRPCBroker, conn *grpc.ClientConn) *pluginClient {
+	var pc pluginClient
+	pc.lggrBroker = &lggrBroker{stopCh: stopCh, lggr: lggr, broker: &pc.atomicBroker}
```

Comment:
i suspect this line is critical, but for the life of me, i don't know why.

the code reads like you set it and then nothing ever uses it. what's going on?

Reply:
Which part are you referring to? All of the methods from the embed types are available and used.
This declaration is just odd because `broker` is an interface and we need to retain a reference to the atomic impl to be able to update it.

I'll update `pluginClient` with a doc.